test/js/blocks.test.js

```diff
@@ -78,20 +78,6 @@ describe("blocks", () => {
     );
   });
 
-  // from ruby test/ruby/test_call.rb
-  test("inline do end", () =>
```

Comment:
Any reason we're removing this test here?

Reply:
sorry, I added it back now.